src/backend/InvenTree/report/api.py

```diff
@@ -261,242 +407,39 @@ def print(self, request, items_to_print):
                 'path': request.path,
             })
 
-    def get(self, request, *args, **kwargs):
-        """Default implementation of GET for a print endpoint.
-
-        Note that it expects the class has defined a get_items() method
-        """
-        items = self.get_items()
-        return self.print(request, items)
-
-
-class StockItemTestReportMixin(ReportFilterMixin):
-    """Mixin for StockItemTestReport report template."""
-
-    ITEM_MODEL = StockItem
-    ITEM_KEY = 'item'
-    queryset = report.models.TestReport.objects.all()
-    serializer_class = report.serializers.TestReportSerializer
-
-
-class StockItemTestReportList(StockItemTestReportMixin, ReportListView):
-    """API endpoint for viewing list of TestReport objects.
-
-    Filterable by:
-
-    - enabled: Filter by enabled / disabled status
-    - item: Filter by stock item(s)
-    """
-
-    pass
-
-
-class StockItemTestReportDetail(StockItemTestReportMixin, RetrieveUpdateDestroyAPI):
-    """API endpoint for a single TestReport object."""
-
-    pass
-
-
-class StockItemTestReportPrint(StockItemTestReportMixin, ReportPrintMixin, RetrieveAPI):
-    """API endpoint for printing a TestReport object."""
-
-    def report_callback(self, item, report, request):
-        """Callback to (optionally) save a copy of the generated report."""
-        if common.models.InvenTreeSetting.get_setting(
-            'REPORT_ATTACH_TEST_REPORT', cache=False
-        ):
-            # Construct a PDF file object
-            try:
-                pdf = report.get_document().write_pdf()
-                pdf_content = ContentFile(pdf, 'test_report.pdf')
-            except TemplateDoesNotExist:
-                return
-
-            StockItemAttachment.objects.create(
-                attachment=pdf_content,
-                stock_item=item,
-                user=request.user,
-                comment=_('Test report'),
-            )
```

Comment:
@SchrodingersGat was this removal intentional? I can not find an obvious replacement in the new code

Reply:
@SchrodingersGat any insights on this?

Reply:
@SchrodingersGat was this intentional? I will add it to the open issues if not